nbclient/_version.py

```diff
@@ -1 +1 @@
-version = '0.3.1'
+version = '0.3.2-dev.0'
```

Comment:
I think this should be 0.4.0-dev.0 because this will break Voilà, which inherits from the executor, which may duplicate the whole Output widget logic.

Reply:
I agree, it changes behavior.